src/navigation/actions.js

```diff
@@ -30,6 +30,17 @@ import { ROUTES } from './constants';
  *
  */
 
+/**
+ * Pushes the Settings page route onto the main navigation stack.
+ */
+export const gotoSettings = () =>
+  NavigationActions.navigate({
+    routeName: ROUTES.SETTINGS,
+    params: {
+      title: 'Settings',
```

Comment:
I was going to ignore, but you did hook up translation strings for the modals of this page. Do the title too?

Reply:
just replying to all localization comments: 
6b517762f559b7b951e4db8260e2e69e7260b4fc
08df2eb70d9437e45a04cf69191e43e64e64263b